python/mujinplanningclient/binpickingplanningclient.py

```diff
@@ -46,641 +75,2936 @@ def __init__(self, regionname=None, **kwargs):
+            taskparameters['atStartPlanDynamicContentsNames'] = atStartPlanDynamicContentsNames
+        if automaticToolPosConstraintNotGrabbing != False:
+            taskparameters['automaticToolPosConstraintNotGrabbing'] = automaticToolPosConstraintNotGrabbing
+        if automaticToolPosConstraintWhenGrabbing != False:
+            taskparameters['automaticToolPosConstraintWhenGrabbing'] = automaticToolPosConstraintWhenGrabbing
+        if barcodeScanningInfo is not None:
+            taskparameters['barcodeScanningInfo'] = barcodeScanningInfo
+        if binpickingDebugMode != 0:
+            taskparameters['binpickingDebugMode'] = binpickingDebugMode
+        if bodyNameToAvoidFinalCollision is not None:
+            taskparameters['bodyNameToAvoidFinalCollision'] = bodyNameToAvoidFinalCollision
+        if bottomScanPlacementInfo is not None:
+            taskparameters['bottomScanPlacementInfo'] = bottomScanPlacementInfo
+        if cameraPlanningInfos is not None:
+            taskparameters['cameraPlanningInfos'] = cameraPlanningInfos
+        if cameraOcclusionApplyGrabbedState != True:
+            taskparameters['cameraOcclusionApplyGrabbedState'] = cameraOcclusionApplyGrabbedState
+        if cameraOcclusionOffset != 20.0:
+            taskparameters['cameraOcclusionOffset'] = cameraOcclusionOffset
+        if cameraOcclusionPaddingTime != 0.1:
+            taskparameters['cameraOcclusionPaddingTime'] = cameraOcclusionPaddingTime
+        if cameraOcclusionPaddingTimeStart != 0.1:
+            taskparameters['cameraOcclusionPaddingTimeStart'] = cameraOcclusionPaddingTimeStart
+        if cameraOcclusionPaddingTimeEnd != 0.1:
+            taskparameters['cameraOcclusionPaddingTimeEnd'] = cameraOcclusionPaddingTimeEnd
+        if cameraOcclusionUseLinkVisibility != True:
+            taskparameters['cameraOcclusionUseLinkVisibility'] = cameraOcclusionUseLinkVisibility
+        if canPlaceInSourceOnRecover != False:
+            taskparameters['canPlaceInSourceOnRecover'] = canPlaceInSourceOnRecover
+        if canPlaceInSourceOnBarcodeScanFail != False:
+            taskparameters['canPlaceInSourceOnBarcodeScanFail'] = canPlaceInSourceOnBarcodeScanFail
+        if canSetIsPickable != True:
+            taskparameters['canSetIsPickable'] = canSetIsPickable
+        if checkCameraOcclusionAtMidDest != False:
+            taskparameters['checkCameraOcclusionAtMidDest'] = checkCameraOcclusionAtMidDest
+        if checkCollisionAtDestNames is not None:
+            taskparameters['checkCollisionAtDestNames'] = checkCollisionAtDestNames
+        if checkDestContainerEmptyOnArrivalNames is not None:
+            taskparameters['checkDestContainerEmptyOnArrivalNames'] = checkDestContainerEmptyOnArrivalNames
+        if checkExpectedDetectedHeightThreshold is not None:
+            taskparameters['checkExpectedDetectedHeightThreshold'] = checkExpectedDetectedHeightThreshold
+        if checkForEndEffectorLowerThanGraspDist != 0.0:
```

Comment:
I think this default is incorrect in the schema. The server defaults to 0.05. The client should use `None`.

Reply:
Done. https://github.com/mujin/mujinplanningclientpy/pull/45/commits/3f9328491272e5d1fa52b4c23fb614b4ab452a1e